setup.py

```diff
@@ -6,6 +6,19 @@
     r"""__version__ = ["']+([0-9\.]*)["']+""",
     open('openaerostruct/__init__.py').read(),
 )[0]
+
+optional_dependencies = {
+    'docs': ['openmdao[docs]==3.9.2'],

```

Comment:
Do we need to pin this version here?

Reply:
Now I specify the docs dependencies (like numpydoc) directly instead of relying on openmdao

Reply:
OpenMDAO has to be 3.9.2 or older because they migrated their docs to jupyter notebooks at 3.10.0. We cannot build the OAS docs with OpenMDAO 3.10.0.

Reply:
Ah okay, makes sense